v/config.yaml

```diff
@@ -1,10 +1,10 @@
 language:
-  version: "weekly.2024.49"
+  version: "latest"
```

Comment:
Could master be considered as stable ?

I mean since v is not at 1.0, it could be ok to point to master, but it could be inconvenient to fix compilation issues (or else) in there is some

Reply:
@waghanza For now, master is as stable as previous. I changed to master because "weekly.2024.49" is breaking the CI.